content/modeling/modeling_4.md

```diff
@@ -0,0 +1,121 @@
++++
+title= "Create Fractribution report table"
+weight = 4
+post = ""
++++
+
+Data from the fractribution dbt package should now be in your warehouse. Next we need to create the report table. There are two ways to to this - by pulling a Docker container image or with Python locally. 
+***
+
+{{< tabs groupId="modeling" >}}
+{{% tab name="Docker" %}}
+To run the fractribution script using Docker, make sure that you have [Docker](https://www.docker.com/products/docker-desktop/) installed.
+
+Once you have docker installed, you can pull the image from the Docker Hub:
+
+```
+docker pull snowplow/fractribution
+```
+
+***
+You can optionally choose a specific attribution model from the following:
+- shapley (default): Takes the weighted average of the marginal contributions of each channel to a conversion.
+- first_touch: Assigns 100% attribution to the first channel in each path.
+- last_touch: Assigns 100% attribution to the last channel in each path.
+- position_based: The first and last channels get 40% of the credit each, with the remaining
+    channels getting the leftover 20% distributed evenly.
+- linear: Assigns attribution evenly between all channels on the path.
+
+| ![Attribution models](../images/attribution_model.png?width=70pc) |
+|:--:|
+| Image source [thinkwithgoogle](https://www.thinkwithgoogle.com/intl/en-ca/marketing-strategies/data-and-measurement/measurement-and-attribution-primer/) |
```

Comment:
The image here does not match the models above. Can we maybe build the same diagram in draw.io or something similar to match the list shown above ?

Reply:
I created a new image, let me know if this was what you had in mind!